test/test_septa_trolley_lines.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+import pendulum
+
+from gtfs_realtime_translators.translators import SeptaTrolleyLineTranslator
```

Comment:
Keep the test file and data. Just use the generic `SwiftlyGtfsRealtimeTranslator`